src/tools/run-make-support/src/fs_wrapper.rs

```diff
@@ -0,0 +1,98 @@
+use std::fs;
+use std::path::Path;
+
+/// A safe wrapper around `std::fs::remove_file` that prints the file path if an operation fails.
+pub fn remove_file<P: AsRef<Path>>(path: P) {
+    fs::remove_file(path.as_ref()).expect(&format!(
+        "the file in path \"{:?}\" could not be removed.",
```

Comment:
Nits:

- I don't think you need the debug format `{:?}` when you already call `display()`? Same for other debug format specifiers.
- Also we usually don't end panic messages with periods (super minor style nit)